### PR TITLE
[Fix] /v2/nodes

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Fix the nodes to return valid information.
- last sent was wrong needed to get max timestamp
- Some of the data was mixed up where:
    - where it moved to
    - missed some sensors

Additionally, it introduces cached data for the expensive api call to look through the SensorData.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: sensorsafrica/management/commands/cache_lastactive_nodes_data.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            return o.timestamp()
+        return super(Encoder, self).default(o)
+
+
+class Command(BaseCommand):
+    help = ""
+
+    def handle(self, *args, **options):
+
+        print(settings.STATIC_ROOT + "/partial_nodes_data.json")
```
Comment (kilemensi) [inline sensorsafrica/management/commands/cache_lastactive_nodes_data.py]: What are we trying to see here?

File: sensorsafrica/api/v2/views.py [hunk 2]
```diff
@@ -207,34 +209,27 @@ class CityView(mixins.ListModelMixin, viewsets.GenericViewSet):
 class NodesView(viewsets.ViewSet):
 
     # Cache requested url for each user for 1 hour
-    @method_decorator(cache_page(3600))
+    # @method_decorator(cache_page(3600))
```
Comment (kilemensi) [inline sensorsafrica/api/v2/views.py]: Remove

File: sensorsafrica/management/commands/cache_lastactive_nodes_data.py (hunk 1 above)
Comment (karimkawambwa) [reply sensorsafrica/management/commands/cache_lastactive_nodes_data.py]: Forgot that log, nice catch.

File: sensorsafrica/api/v2/views.py (hunk 2 above)
Comment (karimkawambwa) [reply sensorsafrica/api/v2/views.py]: nice catch

**Comment by karimkawambwa** [comment]
@kilemensi I switch to having a database table like talked instead of using a json file.

**Pull request merged**